script.js

```diff
@@ -0,0 +1,30 @@
+//Declare and initilize variables
+let howOldAreYou;
+let invalidInput;
+let Age;
+let approvedAge = 21;
+const WELCOME_MESSAGE = `Welcome To The Kosher Wine Cellar!
+You must be over 21 years to purchase on our site!`;
```

Comment:
Since you've defined the minimum age above (`approvedAge`) you can now use it inside the message
```suggestion
const WELCOME_MESSAGE = `Welcome To The Kosher Wine Cellar!
You must be over ${approvedAge} years to purchase on our site!`;
```